build/cluster-operator-ansible/playbooks/cluster-api-prep/files/cluster-api-template.yaml

```diff
@@ -333,3 +384,10 @@ parameters:
 # location of container image
 - name: IMAGE
   value: quay.io/openshift/kubernetes-cluster-api:latest
+# machine controller image
+- name: MACHINE_CONTROLLER_IMAGE
+  value: quay.io/csrwng/aws-machine-controller:latest
```

Comment:
Looks like this should be openshift here.

Reply:
@dgoodwin it's what will work for now. It seems I can no longer create image repos under the openshift org in quay.io

Reply:
Gotcha, that is weird.